TitleCreator.py

```diff
@@ -5,11 +5,12 @@
 
 def ofprefix():
     l = ['Viceroy', 'Commandant', 'Grand Poo-Bah', 'Archon', 'Duke', 'Chancellor', 'President', 'Marquis',
-            'Earl', 'Director', 'Chair', 'Head', 'Senior Director', 'Vice President', 'Dark Lord']
+            'Earl', 'Director', 'Chair', 'Head', 'Senior Director', 'Vice President', 'Dark Lord',
+            'Dread Lord', 'Czar', 'Emperor', 'Baron', 'Guru', 'Administrator']
     return choice(l)
 
 def prefix():
-    l = ['Principal', 'Chief', 'Head', 'Lead', 'Senior', 'Master']
+    l = ['Principal', 'Chief', 'Head', 'Lead', 'Senior', 'Master', 'Arch', 'Grand', 'Premier']
```

Comment:
Arch/Grand don't seem to flow here (arch systems engineer?) but I like Premier. Can you remove the other two from this PR so I can merge it?